### ISSUE TITLE
36095074: fwd/back buttons have been removed from text fields in iOS

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#19092]
#### Description

Summary:

forward/back buttons in iOS text fields have been removed. this exacerbates RSI and makes the larger iphone 6S+ far harder to use for older people or those with disabilities. if you make a mistake whiile tapping a name in, it is no longer possible to skip back and fix a single character by using the large keypad buttons, instead you need fine motor skills to position the caret in exactly the right spot. this is *much* harder to do for partially abled people as a result.

Steps to Reproduce:

- open safari in landscape mode (or any other ios app with text fields present)
- type "fuo bar" by mistake - it should be 'foo bar'

Expected Results:

use fwd/back buttons to fix a single character error quickly and easily with buttons.

Actual Results:

as the skip back button is no longer available, so you need to precisely locate the text input pointer in exactly the right spot to correct a single character. this is really tricky to do it if your finger control isn't perfect, like mine, or for older people, and it very very frustrating. one of the main reasons I got an iphone 6s+ was because of this and the large screen.

Version/Build:

ios 11 and higher (specifically 11.2.1)

Configuration:

iphone 6s+

-
Product Version: 11.2.1
Created: 2017-12-17T13:56:48.685450
Originated: 2017-12-17T00:00:00
Open Radar Link: http://www.openradar.me/36095074
